events/wc_undead_events.txt

```diff
@@ -1147,6 +1156,7 @@ character_event = {
 						is_society_grandmaster = yes
 					}
 				}
+				NOT = { religion = throsic }
```

Comment:
So, there's a problem, being a secret Throsic, I'm not able to create wickerman.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79164260-c5f54f80-7df1-11ea-8bee-e19988ff9312.png)

</details>
So, it should be this instead.

```suggestion
				NOT = { true_religion = throsic }
```

Reply:
@Velius94 not resolved, I'm not able to create wickerman.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79267511-568f6680-7eaa-11ea-9e30-a3b92577848d.png)

</details>